### ISSUE TITLE
Wdym in HTML lol

### ISSUE THREAD

**Issue by DeathlyBower959** [Esxml/vapor-ghost-client#1]

**Comment by Esxml** [comment]
its an external html and js mcpe client for pc

**Comment by DeathlyBower959** [comment]
are you ok

**Comment by Esxml** [comment]
yes and we already have time display and fps counter so dont doubt it and we have a mod menu, launcher and the mod menu opens on toggle

**Comment by Esxml** [comment]
join our discord. you can see sneak peeks! https://discord.gg/YUjha3VJAU
